README.md

```diff
@@ -77,7 +77,17 @@ More precisely:
 
 2. The Core Team will regularly review `swift-evolution` pull requests, and provide feedback to the authors in the pull request on whether or not the proposal looks within reason of something that might be accepted.
 
-3. If a proposal gets a positive indicator from the Core Team for later review, the authors must provide an implementation prior to the proposal being formally reviewed.  An implementation should be provided in the form of a pull request against the impacted repositories (e.g., `swift`, `swift-package-manager`), and the proposal should be updated with a link to that pull request.  The existence of an implementation does not guarantee that the proposal will be accepted, but it is instrumental in evaluating the quality and impact of the proposal.
+3. If a proposal gets a positive indicator from the Core Team for later review, the authors must provide an implementation prior to the proposal being formally reviewed.  An implementation should be provided in the form of a pull request against the impacted repositories (e.g., `swift`, `swift-package-manager`), and the proposal should be updated with a link to that pull request.  It is expected that before review begins the pull request implementation runs at least the following tests:
+
+    a. A full test on Linux and macOS. (e.x. "@swift-ci test").
+
+    b. A full test of the Source Compatibility Suite to identify source breakage. (e.x. "@swift-ci test source compatibility").
+
+    c. A full run of Swift Compiler Performance. (e.x. "@swift-ci test compiler performance").
```

Comment:
We should elaborate on how this data will be presented and used during the review process.  Further, many people don't know how to interpret this data on their own.  I can see this data being useful for assessing a proposal and it's prototype implementation, but a prototype implementation doesn't need to be production quality for a review to be run.